src/main/java/salesforce/ui/pages/lightning/campaign/CreateCampaignPage.java

```diff
@@ -107,4 +114,35 @@ public CampaignCreatedPage clickSaveBtn() {
     public void clickActive() {
         webElementAction.clickByLocator(activeCheck);
     }
+
+    @Override
+    public void fillUpField(Map<String, String> table) {
+        HashMap<String, VoidSupplier> actionsContractMap = mapActionsContract(table);
+        table.keySet().forEach(key -> actionsContractMap.get(key).run());
+    }
+
+    @Override
+    public CampaignCreatedPage clickSaveButton() {
+        webElementAction.clickByLocator(saveButton);
+        return new CampaignCreatedPage();
+    }
+
+    /**
+     * Saves actions in New work type page in map.
+     *
+     * @param campaignMap is map
+     * @return a map with action of fields
+     */
+    private HashMap<String, VoidSupplier> mapActionsContract(final Map<String, String> campaignMap) {
+        HashMap<String, VoidSupplier> mapActions = new HashMap<>();
+        mapActions.put("Campaign Name", () -> setInputField("Campaign Name", campaignMap.get("Campaign Name")));
+        mapActions.put("Start Date", () -> setInputField("Start Date", campaignMap.get("Start Date")));
+        mapActions.put("End Date", () -> setInputField("End Date", campaignMap.get("End Date")));
```

Comment:
This lines are less than 120 chars?
Maybe you could run a checkstyle test, I think there some checkstyle corrections missing

Reply:
ok